hbase-server/src/test/java/org/apache/hadoop/hbase/master/procedure/TestModifyTableProcedure.java

```diff
@@ -584,4 +589,100 @@ public void run() {
     t2.join();
     assertFalse("Expected ConcurrentTableModificationException.", (t1.exception || t2.exception));
   }
+
+  @Test
+  public void testModifyWillNotReopenRegions() throws IOException {
+    final boolean reopenRegions = false;
+    final TableName tableName = TableName.valueOf(name.getMethodName());
+    final ProcedureExecutor<MasterProcedureEnv> procExec = getMasterProcedureExecutor();
+
+    MasterProcedureTestingUtility.createTable(procExec, tableName, null, "cf");
+    UTIL.getAdmin().disableTable(tableName);
```

Comment:
Can we do this without disabling the Table in UT ? As in the actual command line flow when using with alter command, one need not disable the table.

Reply:
This was an oversight. I will remove this code in the next commit.